pylint/extensions/code_style.py

```diff
@@ -36,7 +36,7 @@ class CodeStyleChecker(BaseChecker):
             "Emitted when dictionary values can be replaced by namedtuples or dataclass instances.",
         ),
         "R6102": (
-            "Consider using an in-place tuple%s",
+            "Consider using an in-place tuple instead of list",
             "consider-using-tuple",
             "Emitted when an in-place defined list or set can be "
             "replaced by a slightly faster tuple.",
```

Comment:
We should probably change that text to say that it's only for consistency as we have use-sequence-for-iteration that handle the performance issue.

Reply:
Good catch, must have missed that! I've updated the documentation. Let me know what you think

Reply:
👍